### DISCUSSION TITLE
Questions about beta to GA process, GitHub blog post and macOS 14 beta or GA status?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [actions/virtual-environments#9283] [General questions] [answered]
Hi, :wave: 

The [GitHub Blog, GitHub Actions: macOS 14 (Sonoma) is now available](https://github.blog/changelog/2024-01-30-github-actions-macos-14-sonoma-is-now-available/) says that you can now use the macOS 14 image with GitHub Actions. This repository's readme says that macOS 14 is in _beta_, but the GitHub blog post does _not_ mention the _beta_ status.

I find this confusing. I don't know which is the source of truth. Should I rely on GitHub's blog post, or your repository readme???

So my questions are:

- Is macOS 14 out of beta for `actions/runner-images`?
- What's the process for getting a image out of beta?
- Does the GitHub Blog post denote GA status for this project?

Thank you. :smile:

**Comment by MaksimZhukov** [comment, marked as answer]
Hello @HonkingGoose! 
Sorry for the confusion!
The macOS 14 runner image is available but only as a beta version.
As mentioned in the [documentation](https://github.com/actions/runner-images?tab=readme-ov-file#image-definitions), the goal of a Beta is to identify and fix any potential issues that exist on that image. We would like to gather feedback on an image before it is released to GA.

**Reply by HonkingGoose** [reply]
Thank you for the response! I'll wait until the macOS 14 runner is GA before adding it to the Renovate bot code.
